openlibrary/templates/lib/nav_foot.html

```diff
@@ -48,17 +48,23 @@ <h2>$:_('Help')</h2>
           <a class="footer-icons__github" href="https://github.com/internetarchive/openlibrary">github</a>
         </aside>
       </div>
-      <div id="footer-locale-menu">
-        <h2>$_('Change Website Language')</h2> 
-        <ul id="locale-options">
-          <li><a href="#" lang="cs" data-lang-id="cs" title="$_('Czech')">Čeština (cs)</a> </li>
-          <li><a href="#" lang="de" data-lang-id="de" title="$_('German')">Deutsch (de)</a> </li>
-          <li><a href="#" lang="en" data-lang-id="en" title="$_('English')">English (en)</a></li>
-          <li><a href="#" lang="es" data-lang-id="es" title="$_('Spanish')">Español (es)</a></li>
-          <li><a href="#" lang="fr" data-lang-id="fr" title="$_('French')">Français (fr)</a></li>
-          <li><a href="#" lang="hr" data-lang-id="hr" title="$_('Croatian')">Hrvatski (hr)</a></li>
-          <li><a href="#" lang="te" data-lang-id="te" title="$_('Telugu')">తెలుగు (te)</a></li>
-        </ul>
+      <div class="language-component" id="footer-locale-menu">
+        <details class="lang-details">
+          <summary class="lang-summary">
+            $_('Language')
+          </summary> 
+          <div class="language-dropdown-component">
+            <ul class="dropdown-menu" id="locale-options">
+              <li><a href="#" lang="cs" data-lang-id="cs" title="$_('Czech')">Čeština (cs)</a> </li>
```

Comment:
INstead of # as the href here, could we use something that toggles the language? e.g. ?lang=cs

We can disable the link behaviour with JavaScript using a click handler with ev.preventDefault

Reply:
@jdlrobson It's functional but you will not notice it on localhost. `HTTP_LANG` Cookie will be added and its value will change accordingly.

Reply:
We can also go ahead with your suggestion.

Reply:
The feature uses the following js to set the cookie.
https://github.com/internetarchive/openlibrary/blob/133ee89ae2bbb290d4a805b2be4763906aa06765/openlibrary/plugins/openlibrary/js/ol.js#L159

> I also think it /should/ do something locally to support our testing and development, even if it's not what we do in production. Otherwise the feature will look broken and be confusing. A query string parameter seems like a good fallback regardless.

I completely agree with you but even localhost:8080?lang=fr doesn't show any visual changes.

Reply:
https://user-images.githubusercontent.com/64412143/117581175-2ea81c00-b119-11eb-8373-ce6ed287b5c0.mp4